packages/squiggle-lang/src/rescript/Reducer/Reducer_ErrorValue.res

```diff
@@ -0,0 +1,26 @@
+type errorValue =
```

Comment:
minor note, but maybe this should be renamed ``t``, and the next function, ``toString`` or ``show``?

Reply:
Every major type has a show. However they all use showError. This way I could avoid writing Reducer.ErrorValue.show every time to avoid name conflict... Let's say we disambiguate show using Reducer.ErrorValue.show all the way. But then I am opening ErrorValue in every module not to write things like Reducer.ErrorValue.REArrayIndexNotFound. Opening the module that contains show would make the screen full of warnings. When we cannot open the value, to reduce Reducer.ErrorValue.REArrayIndexNotFound to simple REArrayIndexNotFound , we have to make long type declaration for each and every variable.  
So this breaking of pattern makes the code really easier.

Reply:
That makes sense, thanks+

Reply:
RESOLVED